src/main/scala/gitbucket/core/service/HandleCommentService.scala

```diff
@@ -134,4 +135,18 @@ trait HandleCommentService {
       }
     }
   }
+
+  def deleteCommentByApi(repoInfo: RepositoryInfo, comment: IssueComment, issue: Issue)(
+    implicit context: Context,
+    s: Session
+  ): Option[(Issue, Int)] = context.loginAccount.flatMap { _ =>
+    val deleteResult = deleteComment(comment.issueId, comment.commentId)
```

Comment:
I couldn't find any docs for `deleteComment` here. It returns an `Int` - am I safe to assume that the returned value is the number of records that were deleted in the DB?

If that's the case, then I think making sure it is equal to a non-zero number (or in this case, 1) would be a good idea. Otherwise I don't think there's any need for this `deleteCommentByApi` function to return `Option[(Issue, Int)]`